block-templates/404.html

```diff
@@ -9,7 +9,7 @@ <h2 class="has-text-align-center" style="font-size:clamp(4rem, 40vw, 20rem);font
 <!-- wp:paragraph {"align":"center"} -->
 <p class="has-text-align-center">This page could not be found.</p>
 <!-- /wp:paragraph -->
-
+<div class="msg">Maybe this page moved? Got deleted? Is hiding out in quarantine? Never existed in the first place?<p>Let's go <a href="#">home</a> and try from there.</p></div>
```

Comment:
```suggestion
<div class="msg">
    <p>Maybe this page moved, got deleted or never existed in the first place?</p>
    <p>Let's go <a href="/">home</a> and try from there.</p>
</div>
```

I think this should be done more like this - with 2 paragraphs and a real link to home.

Reply:
It can be changed anytime. Thank you for your suggestion.